index.html

```diff
@@ -1265,9 +1265,7 @@
 
 [=User agents=] should aggressively <a href="#data-minimization">minimize</a> [=ancillary
 data=] and should avoid burdening the user with additional [=privacy labor=]
-when deciding what [=ancillary data=] to expose. To that end, user agents may
-employ user research, solicitation of general preferences, and heuristics about
-sensitivity of data or trust in a particular context. To facilitate site
+when deciding what [=ancillary data=] to expose. To facilitate site
```

Comment:
how about simpler language? "To help sites understand user preferences..."

Reply:
I think this is a good change too